src/debian/manifest.json

```
@@ -194,6 +194,20 @@
             }
           ]
         },
+        {
+          "platforms": [
+            {
+              "architecture": "arm64",
+              "dockerfile": "src/debian/11/opt/arm64v8",
+              "os": "linux",
+              "osVersion": "bullseye",
+              "tags": {
+                "debian-11-helix-arm64v8-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
```

This tag name conflicts the entry above.

```suggestion
                "debian-11-opt-arm64v8-$(System:TimeStamp)-$(System:DockerfileGitCommitSha)": {}
```